core/schema/container.graphqls

```diff
@@ -505,6 +505,20 @@ type Container {
     path: String!
   ): Container!
 
+  """
+  Indicate that subsequent commands should be featured more
+  prominently in the UI.
+  """
+  withFocus: Container!
+
+  """
+  Indicate that subsequent commands should not be featured
```

Comment:
nit: would it be more technically correct and explicit to say `subsequent withExecs` instead of `commands`?

`withExecs` is probably a very awkward adverb(?), so maybe just `subsequent execs`?

Just don't want to create ambiguity around the term "commands".

Reply:
wdyt of "operations"? Right now it's only execs, but we might want to support something like `foo.WithFocus().Export(ctx)` too (assuming `Export` does something useful like printing filenames as it exports).

Reply:
Yeah operations sounds good!